TS/translations/ts.pot

```diff
@@ -3,13 +3,13 @@
 # This file is distributed under the same license as the PACKAGE package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
-#: document.h:232
```

Comment:
yeah this file changes all the time.. its just in the comments, I think it doesn't need updating as much